src/core_ocean/analysis_members/mpas_ocn_mixed_layer_depths.F

```diff
@@ -216,9 +221,16 @@ subroutine ocn_compute_mixed_layer_depths(domain, timeLevel, err)!{{{
          call mpas_pool_get_subpool(block % structs, 'state', statePool)
          call mpas_pool_get_subpool(block % structs, 'mesh', meshPool)
          call mpas_pool_get_subpool(block % structs, 'diagnostics', diagnosticsPool)
+         call mpas_pool_get_subpool(block % structs, 'forcing', forcingPool)
          call mpas_pool_get_subpool(block % structs, 'mixedLayerDepthsAM', mixedLayerDepthsAMPool)
          call mpas_pool_get_subpool(statePool, 'tracers', tracersPool)
 
+         !Use work array for storing of pressure, potentially modified by landIce
+         call mpas_pool_get_subpool(block % structs, 'mixedLayerDepthsAMScratch', scratchPool)
+         call mpas_pool_get_field(scratchPool, 'pressureAdjustedForLandIceScratch', pressureAdjustedForLandIceField)
```

Comment:
@vanroekel, I'm getting an error: "forrtl: severe (71): integer divide by zero" from this line.

We should double check based on a different AM that uses a scratch variable that we did this correctly here.

Reply:
This was caused by the fact that the Registry has `Depth` but the subpool call above is with `Depths` (plural).  The pool didn't exist and therefor had zero size and gave a divide by zero error.

Re-testing with this fix.